srcalc/src/main/java/gov/va/med/srcalc/db/RiskModelDao.java

```diff
@@ -33,12 +38,30 @@ protected Session getCurrentSession()
      * Returns all RiskModels in the database, in arbitrary order.
      * @return an ImmutableCollection
      */
-    public ImmutableCollection<RiskModel> getAllRiskModels()
+    @SuppressWarnings("unchecked")
```

Comment:
This annotation can go specifically on line 45 instead of on the whole method. It is best to keep warnings suppression as isolated as possible.